scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -3168,12 +3171,15 @@ public List<JobLabelInfo> getLabels() {
         });
     }
 
-    public List<JobLabelInfo> newLabels(List<String> labels) throws LabelConflictException {
+    public List<JobLabelInfo> newLabels(List<String> labels) throws LabelConflictException, LabelValidationException {
         List<JobLabelInfo> jobLabelsInfo = new LinkedList<>();
         for (String label : labels) {
             if (checkIfLabelExists(label)) {
                 throw new LabelConflictException(label);
             }
+            if (label.length() > LABEL_MAX_LENGTH.getValueAsInt() || !label.matches(LABEL_REGEX.getValueAsString())) {
```

Comment:
as PASchedulerProperties uses lazy loading and a configuration file, it's not a good practice to use import static.

It might work, but I would prefer if you use it directly with e.g. PASchedulerProperties.LABEL_MAX_LENGTH.getValueAsInt()